### PR TITLE
Update README.md for eqn2mml

### PR DESCRIPTION
**Pull request by ualiangzhang**
Update the README for eqn2mml

### CODE REVIEW

**Comment by ualiangzhang** [comment]
@myedibleenso I updated the README for the updated eqn2mml service. Could you help review it? Thanks!

File: skema/img2mml/README.md [hunk 1]
```diff
@@ -22,10 +26,10 @@ If you have the checkpoint in the `trained_models` directory already and hope to
 
 To update the model name or path, please make the following modifications to support updating the img2mml service and the corresponding Docker operations:
 
-1. Modify the paths of the model_path variable in the get_mathml_from_bytes function in the skema/img2mml/api.py file.
-2. Update the path settings in the "retrieve latest model for img2mml component" section of skema/.github/workflows/docs.yml.
-3. Adjust the curl command in the test_equation_reading section of skema/.drone.yml to download the checkpoint.
-4. Update the download checkpoint path in skema/img2mml/README.md.
+1. Modify the paths of the model_path variable in the `skema/img2mml/eqn2mml.py` file.
```
Comment (myedibleenso) [inline skema/img2mml/README.md:29]: Why don't we use an ENV variable for this instead?

File: skema/img2mml/README.md [hunk 2]
```diff
@@ -39,15 +43,15 @@ To test the service without Docker (e.g., for development purposes), ensure
...
+Then, run the following command to launch the `eqn2mml` server program, including the `img2mml` and `latex2mml` services:
 
 ```
-uvicorn skema.img2mml.img2mml:app --reload
+uvicorn skema.img2mml.eqn2mml:app --reload
 ```
 
-An example test program is provided as well, which you can invoke with:
-Make sure that you are in generate_mathml folder.
+Unit tests are provided as well, which you can find them in the `tests` directory folder:
```
Comment (myedibleenso) [inline skema/img2mml/README.md:52]: runnable using the following invocation:

```bash
pytest skema/img2mml/tests
```

**Review by myedibleenso — CHANGES_REQUESTED** [review]
@ualiangzhang , I think we should make `model_path` set via an ENV variable (ex. `SKEMA_IMG2MML_MODEL_PATH`, with your default)

**Comment by ualiangzhang** [comment]
@myedibleenso Updated readme and code according to your suggestions. Thanks!

**Review by myedibleenso — APPROVED** [review]

**Pull request merged**